frontend/ui/widget/filechooser.lua

```diff
@@ -415,6 +426,15 @@ function FileChooser:showSetProviderButtons(file, filemanager_instance, reader_u
                 UIManager:close(self.set_provider_dialog)
             end,
         },
+        {
+            text = _("Reset Default"),
```

Comment:
```suggestion
            text = _("Reset default"),
```

Reply:
Might want to have the buttons on 2 lines, eg.:
<kbd>![image](https://user-images.githubusercontent.com/24273478/67041974-1d5a8180-f127-11e9-80c5-1d66a1f39fc4.png)</kbd>

Reply:
i find two lines harder to press, personally

Reply:
also that would require reworking the whole `open as` widget which i am trying to leave as intact as possible.

Reply:
> i find two lines harder to press, personally

Yes, me too. See https://github.com/koreader/koreader/issues/5449#issuecomment-537935889. And we should try to get that better.

But for consistency, and because, if these 3 text fit without truncation in english, they may get truncated in german or in some other language, so it should be on 3 lines.

> also that would require reworking the whole 

No, it should be easy.
Instead of 
```lua
table.insert(buttons, { -- line 1
 {button1}, -- columns
 {button2},
 {button3},
})
```
use
```lua
table.insert(buttons,
 { -- line1
  {button1}, -- columns
  {button2},
 },
 { -- line1
  {button3},
 }
)
```

(Not familiar enough with DocumentRegistry to comment more on this PR.)

Reply:
Or better, if you can detect the current value is different than the default, just add that 2nd row with the `Use default` when it is - so in normal conditions, we only see a single row with the 2 buttons.
```lua
table.insert(buttons, { -- line 1
 {button1}, -- columns
 {button2},
})
if there_is_setting_other_than_default then
    table.insert(buttons, { -- line 2
     {button3},
    })
end
```

Reply:
fixed